spacy_transformers/layers/hf_shim.py

```diff
@@ -0,0 +1,95 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Any
+import srsly
+from functools import partial
+import torch
+from spacy.vectors import get_current_ops
+
+from ..util import make_tempdir
+
+from thinc.api import PyTorchShim
+
+from transformers import AutoModel, AutoConfig, AutoTokenizer
+
+
+class HFModel:
+    def __init__(
+        self,
+        transformer,
+        tokenizer,
+        tokenizer_config: dict = {},
+        transformer_config: dict = {},
+    ):
+        self.transformer = transformer
+        self.tokenizer = tokenizer
+        self.tokenizer_config = tokenizer_config
+        self.transformer_config = transformer_config
```

Comment:
Probably dangerous to assign directly from the value, given that we're using a mutable default:

```suggestion
        self.tokenizer_config = dict(tokenizer_config)
        self.transformer_config = dict(transformer_config)
```